rpc: TestClockOffsetInPingRequest failed

rpc.TestClockOffsetInPingRequest [failed](https://mesolite.cluster.engflow.com/invocations/default/b944ace7-b8cc-4bf3-a384-2ec65ef0a2aa?testReportRun=7&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvcnBjOnJwY190ZXN0) on release-25.3 @ [9569f6f91e386f4200c2d82d46361722b97ca66c](https://github.com/cockroachdb/cockroach/commits/9569f6f91e386f4200c2d82d46361722b97ca66c):


```
=== RUN   TestClockOffsetInPingRequest
    context_test.go:229: server listen
    context_test.go:236: client dial
E251005 04:20:21.717432 456 2@rpc/peer.go:675  [rnode=1,raddr=127.0.0.1:35755,class=system,rpc] 10  failed connection attempt (last connected 0s ago): operation "conn heartbeat" timed out after 241ms (given timeout 200ms): grpc: context deadline exceeded [code 4/DeadlineExceeded]
I251005 04:20:21.722348 456 2@rpc/peer.go:601  [rnode=1,raddr=127.0.0.1:35755,class=system,rpc] 11  connection is now healthy (after 0s)
    context_test.go:250: first two pings check
    context_test.go:258: ping 1 check
    context_test.go:266: 
        	Error Trace:	pkg/rpc/context_test.go:266
        	            				pkg/rpc/context_test.go:181
        	Error:      	Should not be zero, but was 0
        	Test:       	TestClockOffsetInPingRequest
        	Messages:   	1
--- FAIL: TestClockOffsetInPingRequest (0.36s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=7</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestClockOffsetInPingRequest.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-55156

--- a/pkg/rpc/context_test.go
+++ b/pkg/rpc/context_test.go
@@ -193,6 +193,7 @@ func TestClockOffsetInClientPingRequest(t *testing.T) {
 }
 
 func testClockOffsetInPingRequestInternal(t *testing.T, clientOnly bool) {
+	skip.UnderDuress(t, "https://github.com/cockroachdb/cockroach/issues/154839")
 	ctx := context.Background()
 	stopper := stop.NewStopper()
 	defer stopper.Stop(ctx)

